src/main/resources/hazelcast-config-schema.json

```diff
@@ -1373,20 +1373,535 @@
+        },
+        "reuse-address": {
+          "type": "boolean",
+          "default": false,
+          "description": "When you shutdown a cluster member, the server socket port will be in the TIME_WAIT state for the next couple of minutes. If you start the member right after shutting it down, you may not be able to bind it to the same port because it is in the TIME_WAIT state. If you set reuse-address to true, the TIME_WAIT state is ignored and you can bind the member to the same port again."
+        }
+      }
+    },
+    "AdvancedNetwork": {
+      "additionalProperties": false,
+      "description": "The advanced network configuration provides fine-tuning capabilities for configuring network with the ability of defining separate server sockets for different protocols, such as for member-member and member-client communication. This configuration renders exclusive or relation with the generic \"network\" configuration element.",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "default": false,
+          "description": "Indicates whether the advanced network configuration is enabled or not."
+        },
+        "join": {
+          "type": "object",
+          "description": "This configuration lets you choose a discovery mechanism that Hazelcast will use to form a cluster. Hazelcast can find members by multicast, TCP/IP lists and by various discovery mechanisms provided by different cloud APIs.",
+          "additionalProperties": false,
+          "properties": {
+            "multicast": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "enabled": {
+                  "type": "boolean",
+                  "default": true,
+                  "description": "Specifies whether multicast discovery is enabled."
+                },
+                "loopbackModeEnabled": {
+                  "type": "boolean",
+                  "default": false,
+                  "description": "Specifies if the loopback mode is enabled for multicast discovery."
+                },
+                "multicast-group": {
+                  "type": "string",
+                  "default": "224.2.2.3",
+                  "description": "Specifies the multicast group IP address when you want to create clusters within the same network."
+                },
+                "multicast-port": {
+                  "type": "integer",
+                  "default": 54327,
+                  "description": "Specifies the multicast socket port that the Hazelcast member listens to and sends discovery messages through."
+                },
+                "multicast-time-to-live": {
+                  "type": "integer",
+                  "minimum": 0,
+                  "maximum": 255,
```

Comment:
Reading multicast config, the default is 2 , please add it

Reply:
done